src/bin/solang.rs

```diff
@@ -878,19 +858,22 @@ fn imports_arg(matches: &ArgMatches) -> FileResolver {
         }
     }
 
-    if let Some(maps) = matches.get_many::<String>("IMPORTMAP") {
-        for p in maps {
-            if let Some((map, path)) = p.split_once('=') {
-                if let Err(e) = resolver.add_import_map(OsString::from(map), PathBuf::from(path)) {
-                    eprintln!("error: import path '{}': {}", path, e);
-                    std::process::exit(1);
-                }
-            } else {
-                eprintln!("error: import map '{}': contains no '='", p);
+    if let Some(maps) = matches.get_many::<(String, String)>("IMPORTMAP") {
+        for (map, path) in maps {
+            if let Err(e) = resolver.add_import_map(OsString::from(map), PathBuf::from(path)) {
+                eprintln!("error: import path '{}': {}", path, e);
                 std::process::exit(1);
             }
         }
     }
 
     resolver
 }
+
+fn parse_import_map(map: &str) -> Result<(String, String), String> {
```

Comment:
This function deserves a comment. I see you use it in `ValueParser::new(parse_import_map)`, but It is not clear what these three returned strings represent. If someone were to re-use your function, they would need to know what it does.

Reply:
Good point. I've added a comment